v2/client/src/helpers/logout.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const logout = () => {
+  axios
+    .get('/api/logout')
+    .then(() => (window.location = '/'))
```

Comment:
I think it would be better to use the redux here, we can make a logout action and use the `history` instead of window.location
as window.location will request all the app files from the server again which will take some time
the reducer needs to reset the auth object to the initial auth object, what do you think @thejoefriel ?